app/views/selectFile.html

```diff
@@ -6,7 +6,7 @@
     <div class="form-group-horizontal">
         <p class="note-required">* indicates required field</p>
 
-        <p ng-if="errors.global" class="page-errors">{{errors.global}}</p>
+        <p ng-if="errors.global" class="page-errors" role="alert">{{errors.global}}</p>
```

Comment:
Yup. I searched for other uses of errors.global and didn't find any. Should have looked for the className instead. Fixed in 650e8bd